Git2/Test.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Git2
+{
+    public class Test
+    {
+        string name = "John";
```

Comment:
string name = "John";
```suggestion
         string name = string.Empty;
```
public Test()
         {
             Console.WriteLine($"Hello {name}");
         }
```suggestion
      public Test()
         {
         name = "John";
             Console.WriteLine($"Hello {name}");
         }
```